plugins/security/nexus-ldap-plugin/src/test/java/org/sonatype/security/realms/ldap/internal/ConnectionBlackListTest.java

```diff
@@ -29,19 +30,26 @@
 
 @Ignore("Test has not been run in a while, disabled in old surefire config")
```

Comment:
There are 2 tests ignored (but updated with surrounding code changes), will see to either enable them or simply drop them.